metriken-exposition/src/parquet.rs

```diff
@@ -113,11 +115,13 @@ impl ParquetSchema {
         if self.metadata.is_empty() && !snapshot.metadata.is_empty() {
             self.metadata = snapshot.metadata;
         }
+
```

Comment:
I'm a little confused as to where the metadata from the metrics is actually being stored in the `ParquetSchema` -- maybe I'm missing it but I don't see it anywhere? I'd expect the metadata to be updated in the loops just above this?

Reply:
I think you're commenting on the file-level metadata which is simply taken from the first snapshot with metadata to add to the file. This is the section that captures the source program and version.

Reply:
Aargh, it is hard to add comments on unedited lines of code. I meant that I'd expect the per-column metadata (not table-level) to be inserted into the counters/gauges/histograms BTreeMaps in this block:

```
       for counter in counters {
            self.counters.entry(counter.name).or_default();
        }

        for gauge in gauges {
            self.gauges.entry(gauge.name).or_default();
        }

        for h in histograms {
            self.histograms.entry(h.name).or_default();
        }
```